erpnext/controllers/accounts_controller.py

```diff
@@ -829,8 +829,8 @@ def validate_currency(self):
 				party_account_currency = get_party_account_currency(party_type, party, self.company)
 
 				if (party_account_currency
-						and party_account_currency != self.company_currency
-						and self.currency != party_account_currency):
+						and (self.currency != party_account_currency
+						or self.currency != self.company_currency)):
```

Comment:
@AfshanKhan I don' think this is correct. A company having Default currency as INR can make invoices in UDS. This will block those. Let's discuss the issue again and reopen this.